felix/bpf/conntrack/inforeader_test.go

```diff
@@ -129,14 +129,21 @@ var _ = Describe("BPF Conntrack InfoReader", func() {
 		),
 		Entry("reverse entry - NAT - expired - swapped legs",
 			conntrack.NewKey(123, backendIP, backendPort, clientIP, clientPort),
-			conntrack.NewValueNATReverse(now-time.Hour, v3.FlagSrcDstBA, LegDstSrc, LegSrcDst, net.IPv4(0, 0, 0, 0), svcIP, svcPort),
+			conntrack.NewValueNATReverse(now-time.Hour, v4.FlagSrcDstBA, LegDstSrc, LegSrcDst, net.IPv4(0, 0, 0, 0), svcIP, svcPort),
 			collector.ConntrackInfo{
 				Expired:      true,
 				IsDNAT:       true,
 				Tuple:        makeTuple(clientIP, backendIP, clientPort, backendPort, 123),
 				PreDNATTuple: makeTuple(clientIP, svcIP, clientPort, svcPort, 123),
 			},
 		),
+		Entry("32-bit flags RW",
+			conntrack.NewKey(123, clientIP, clientPort, backendIP, backendPort),
+			conntrack.NewValueNormal(now, v4.FlagSeventeenthBit, LegSrcDst, LegDstSrc),
```

Comment:
I think you could do without explicitly naming the bit

Reply:
That has to come out alright - thx